app/views/spam2/_queue.html.erb

```diff
@@ -2,11 +2,7 @@
 $(document).ready(function () {
 	var table = table_main("#queue_table");
 	$("#all").click(function () { //select all button
-		$('.selectedId').prop('checked', !$('.selectedId').prop('checked'));
-		var check = ($('.selectedId').filter(":checked").length == $('.selectedId').length);
-		$('#select-count').text($('.selectedId').filter(":checked").length);
-		$('#selectall').prop("checked", check);
-		disable_buttons('#selectall');
+  select_all();
```

Comment:
Hi @AmenuveveDordor, can you indent line 5 so that its on the same line as line 8? Thanks!

Reply:
working on it right away


> Hi @AmenuveveDordor, can you indent line 5 so that its on the same line as line 8? Thanks!

Reply:
@TildaDares I have made the changes requested. Thank you for your feedback.